lua/lvim/core/cmp.lua

```diff
@@ -329,10 +242,15 @@ M.config = function()
             confirm_opts.behavior = cmp.ConfirmBehavior.Insert
           end
           if cmp.confirm(confirm_opts) then
+            if luasnip.locally_jumpable() then
+              luasnip.jump()
+            end
```

Comment:
I think this should be in callback of `cmp.confirm`, it overwrites parenthesis for me